### ISSUE TITLE
Create release?

### ISSUE THREAD

**Issue by eitsupi** [quarto-dev/quarto-actions#21]
It is recommended that v2 releases be created because dependabot, which is widely used for automatic GHA updates, has the ability to automatically include release notes in PullRequests.

**Comment by cscheid** [comment]
Yup, it's in our plans to do that in the next couple of days, thank you. We need to finish reviewing the documentation but it's coming for sure.
